receiver/jmxreceiver/config.go

```diff
@@ -11,55 +11,81 @@ import (
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	Endpoint string `mapstructure:"endpoint"`
 	// The target system for the metric gatherer whose built in groovy script to run.
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	TargetSystem string `mapstructure:"target_system"`
 	// The duration in between groovy script invocations and metric exports (10 seconds by default).
 	// Will be converted to milliseconds.
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	CollectionInterval time.Duration `mapstructure:"collection_interval"`
-	// The exporter settings for
+	// The OTLP exporter settings
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	OTLPExporterConfig otlpExporterConfig `mapstructure:"otlp"`
 	// The JMX username
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	Username string `mapstructure:"username"`
 	// The JMX password
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	Password configopaque.String `mapstructure:"password"`
 	// The keystore path for SSL
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	KeystorePath string `mapstructure:"keystore_path"`
 	// The keystore password for SSL
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	KeystorePassword configopaque.String `mapstructure:"keystore_password"`
 	// The keystore type for SSL
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	KeystoreType string `mapstructure:"keystore_type"`
 	// The truststore path for SSL
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	TruststorePath string `mapstructure:"truststore_path"`
 	// The truststore password for SSL
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	TruststorePassword configopaque.String `mapstructure:"truststore_password"`
 	// The truststore type for SSL
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	TruststoreType string `mapstructure:"truststore_type"`
 	// The JMX remote profile.  Should be one of:
 	// `"SASL/PLAIN"`, `"SASL/DIGEST-MD5"`, `"SASL/CRAM-MD5"`, `"TLS SASL/PLAIN"`, `"TLS SASL/DIGEST-MD5"`, or
 	// `"TLS SASL/CRAM-MD5"`, though no enforcement is applied.
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	RemoteProfile string `mapstructure:"remote_profile"`
 	// The SASL/DIGEST-MD5 realm
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	Realm string `mapstructure:"realm"`
 	// Array of additional JARs to be added to the class path when launching the JMX Metric Gatherer JAR
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	AdditionalJars []string `mapstructure:"additional_jars"`
 	// Map of resource attributes used by the Java SDK Autoconfigure to set resource attributes
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	ResourceAttributes map[string]string `mapstructure:"resource_attributes"`
 	// Log level used by the JMX metric gatherer. Should be one of:
 	// `"trace"`, `"debug"`, `"info"`, `"warn"`, `"error"`, `"off"`
+	// Supported by: jmx-metric-gatherer
```

Comment:
If needed, we could add support for this in the jmx-scraper later, if it's set and ignored there is no harm so it's fine for now and there is no urgency to add it.

Reply:
If log_level is set and a jmx-scraper JAR is defined, the receiver will return the following error message on config validation: `"`log_level` can only be used with a JMX Metrics Gatherer JAR"`

Reply:
jmx-scraper does not allow to set the log level without using an external properties file as it uses the JUL framework. For not it should probably be simpler to ignore this option for JMX scraper.

as a side-note, do you know if the scraper/gatherer logs are also captured and sent to the collector ? If yes, then it could maybe be simpler to make jmx-scraper send directly its own logs to the otlp endpoint.

Reply:
Yes, the collector captures the stdout output of the jmx-scraper process and uses the zap.Logger to output them (as receiver/collector logs). Only metrics are propagated down the pipeline.